Library/src/main/java/com/shopify/testify/internal/processor/ParallelPixelProcessor.kt

```diff
@@ -55,7 +55,9 @@ class ParallelPixelProcessor private constructor() {
             launch(executorDispatcher) {
                 (0 until chunkData.chunks).map { chunk ->
                     async {
-                        for (i in (chunk * chunkData.chunkSize) until ((chunk + 1) * chunkData.chunkSize)) {
```

Comment:
This was the bug here. You can see the `end` condition was always using a multiple of `chunkData.chunkSize`, so if the last chunk was less than `chunkSize`, you'd run over the buffer